lib/vty.c

```diff
@@ -3075,9 +3075,20 @@ DEFUN (log_commands,
        log_commands_cmd,
        "log commands",
        "Logging control\n"
-       "Log all commands (can't be unset without restart)\n")
+       "Log all commands\n")
 {
-	do_log_commands = 1;
+	do_log_commands = true;
+	return CMD_SUCCESS;
+}
+
+DEFUN (no_log_commands,
```

Comment:
You could do `[no] log commands` and
```
do_log_commands = (strmatch(argv[0]->text, "no"))
```
to save us the new command here

Reply:
I actually thought about adding a DEFPY but it just seemed like more work than necessary.  I also thought we couldn't combine `'no' with DEFUNS?

Reply:
no, that restriction was lifted a while ago